### PR TITLE
Feat#38/change material-ui version to mui 5

### PR DESCRIPTION
**Pull request by MounikaReddy15**
## Purpose
_change material-ui version to mui 5_

## Ticket number
#38

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import 'react-notifications/lib/notifications.css';
 import { hot } from 'react-hot-loader';
 import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { ThemeProvider } from '@material-ui/core/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
```
Comment (RadhikaPPatel) [inline src/App.js:6]: Why do we need StyledEngineProvider?

File: src/utils/mediaQuery.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-import { useTheme } from '@material-ui/core/styles';
-import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
 
 // TODO: Convert to provider
 export const isMobile = () => {
   const theme = useTheme();
-  return useMediaQuery(theme.breakpoints.down('sm'));
+  return useMediaQuery(theme.breakpoints.down('md'));
```
Comment (RadhikaPPatel) [inline src/utils/mediaQuery.js:8]: Why is this changed from sm to md?

File: src/components/Alert/Alert.js [hunk 3]
```diff
@@ -35,9 +35,11 @@ function Alert({ data, dispatch }) {
         anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
       >
         {data && (
-          <AlertData onClose={handleClose} severity={data.type}>
-            {data.message}
-          </AlertData>
+          <div>
```
Comment (RadhikaPPatel) [inline src/components/Alert/Alert.js:38]: Please refer to TP changes for this particular component.

File: src/layout/TopBar/TopBar.js [hunk 4]
```diff
@@ -55,12 +91,33 @@ function TopBar({
...
+          {isSuperAdmin && (
+          <TextField
+            className={classes.globalFilter}
+            variant="outlined"
+            fullWidth
+            id="org"
+            label="Organization"
+            select
+          >
+            <MenuItem>
```
Comment (RadhikaPPatel) [inline src/layout/TopBar/TopBar.js:103]: Why is only None option hard coded here for Super Admin?

File: src/layout/TopBar/TopBar.js [hunk 5]
```diff
@@ -55,12 +91,33 @@ function TopBar({
         </Link>
 
         <div className={classes.menuRight}>
-          <Link to={routes.USER_MANAGEMENT}>
-            <IconButton aria-label="user-management" color="inherit">
-              <GroupIcon fontSize="large" className={classes.menuIcon} />
-            </IconButton>
-          </Link>
-          <IconButton aria-label="logout" color="inherit" onClick={handleLogoutClick}>
+          {isSuperAdmin && (
```
Comment (RadhikaPPatel) [inline src/layout/TopBar/TopBar.js:94]: Check the placement as well. It is all disturbed.

image

File: src/pages/UserManagement/UserGroups/UserGroups.js [hunk 6]
```diff
@@ -1,17 +1,22 @@
+/* eslint-disable import/no-named-as-default-member */
```
Comment (RadhikaPPatel) [inline src/pages/UserManagement/UserGroups/UserGroups.js:1]: Please remove these ignore and resolve the issues in the file.

File: src/pages/UserManagement/UserManagement.js [hunk 7]
```diff
@@ -161,8 +166,14 @@ function UserManagement({ dispatch, loading, error, user, history, location }) {
   );
 }
 
-const mapStateToProps = (state, ownProps) => ({
-  ...ownProps,
-  ...state.authReducer,
-});
+const mapStateToProps = (state, ownProps) => {
+  const loaded = state.coreuserReducer.loaded && state.coregroupReducer.loaded;
```
Comment (RadhikaPPatel) [inline src/pages/UserManagement/UserManagement.js:170]: Add authreducer loaded check as well

File: src/pages/UserManagement/Users/Users.js [hunk 8]
```diff
@@ -1,19 +1,28 @@
+/* eslint-disable import/no-named-as-default-member */
```
Comment (RadhikaPPatel) [inline src/pages/UserManagement/Users/Users.js:1]: Remove these ignores and fix the file issues.

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Please fix the requested things/issues

File: src/layout/TopBar/TopBar.js (hunk 5 above)
Comment (MounikaReddy15) [reply src/layout/TopBar/TopBar.js:94]: Radhika can you tell me screen resolution, for me all medium and large screens placement is proper

File: src/layout/TopBar/TopBar.js (hunk 4 above)
Comment (MounikaReddy15) [reply src/layout/TopBar/TopBar.js:103]: Because we don't have any api to fetch all org names

Comment (RadhikaPPatel) [reply src/layout/TopBar/TopBar.js:103]: There should be an API that we already use in TP as well as Insights code. So it should be similar here as well.

File: src/layout/TopBar/TopBar.js (hunk 5 above)
Comment (RadhikaPPatel) [reply src/layout/TopBar/TopBar.js:94]: 1320 x 1000

Comment (MounikaReddy15) [reply src/layout/TopBar/TopBar.js:94]: image

File: src/App.js (hunk 1 above)
Comment (MounikaReddy15) [reply src/App.js:6]: It gives custom styles precedence over your MUI

File: src/layout/TopBar/TopBar.js (hunk 4 above)
Comment (MounikaReddy15) [reply src/layout/TopBar/TopBar.js:103]: I tried and it was throwing 404 confirmed with BE as well that there's no API in buildly.io to fetch all orgs

Comment (RadhikaPPatel) [reply src/layout/TopBar/TopBar.js:103]: If we want a global admin filter we need to have a list of organizations. Without it, you cannot create a dropdown for global admin.

**Review by RadhikaPPatel — APPROVED** [review]
Looks good
